device/common/include/traccc/finding/device/impl/propagate_to_next_surface.ipp

```diff
@@ -40,18 +38,13 @@ TRACCC_DEVICE inline void propagate_to_next_surface(
     // Seed id
     unsigned int orig_param_id = links.at(globalIndex).seed_idx;
 
-    // Navigation candidate buffer
-    vecmem::jagged_device_vector<typename propagator_t::intersection_type>
-        nav_candidates(nav_candidates_buffer);
-
     // Count the number of tracks per seed
     vecmem::device_atomic_ref<unsigned int> num_tracks_per_seed(
         n_tracks_per_seed.at(orig_param_id));
 
     const unsigned int s_pos = num_tracks_per_seed.fetch_add(1);
 
-    if (s_pos >= cfg.max_num_branches_per_seed ||
-        globalIndex >= nav_candidates.size()) {
+    if (s_pos >= cfg.max_num_branches_per_seed || globalIndex >= n_in_params) {
```

Comment:
I don't believe this is correct; need to look into this.

Reply:
Removed